server.js

```diff
@@ -1,27 +1,64 @@
-import express from "express";
+import express, { response } from "express";
 import cors from "cors";
+import booksData from "./data/books.json";
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
 
 // Defines the port the app will run on. Defaults to 8080, but can be overridden
 // when starting the server. Example command to overwrite PORT env variable value:
 // PORT=9000 npm start
 const port = process.env.PORT || 8080;
 const app = express();
 
+
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+// Route One - main "page"
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send("Hello, let's check out the world of BOOKS!")
+});
+
+// Route Two - All books with query params (title and average_rating)
+app.get("/books", (req, res) => {
+  const { authors, average_rating } = req.query
+  let books = booksData
+  if (average_rating) {
+    books = booksData.filter(rating => rating.average_rating.toLowerCase() === average_rating.toLowerCase());
+  }
+  if (authors) {
+    books = booksData.filter(name => { return name.authors.toLowerCase() === authors.toLowerCase()});
+  }
+
```

Comment:
An alternative is to use `include.` instead of `===`, which would make it possible to write out only parts of the author's name and still get the needed results from the API. 

For example:   
```
if (authors) {
    books = booksData.filter((name) =>
    name.authors.toLowerCase()
    .includes(authors.toLowerCase()))
```